tests/ui/test_ui_tools.py

```diff
@@ -2251,13 +2251,40 @@ def test_keypress_STREAM_MESSAGE(
             "msg_body_edit_enabled",
             "msg_body_edit_limit",
             "expect_editing_to_succeed",
+            "expected_footer_text_stream",
+            "expected_footer_text_private",
         ],
         [
-            ({"sender_id": 2, "timestamp": 45}, True, True, 60, False),
-            ({"sender_id": 1, "timestamp": 1}, True, False, 60, True),
-            ({"sender_id": 1, "timestamp": 45}, False, True, 60, False),
-            ({"sender_id": 1, "timestamp": 45}, True, True, 60, True),
-            ({"sender_id": 1, "timestamp": 1}, True, True, 0, True),
+            (
+                {"sender_id": 2, "timestamp": 45},
+                True,
+                True,
+                60,
+                False,
+                " You can't edit messages sent by other users.",
+                " You can't edit messages sent by other users.",
+            ),
+            (
+                {"sender_id": 1, "timestamp": 1},
+                True,
+                False,
+                60,
+                True,
+                " Only topic editing allowed."
+                " Time Limit for editing the message body has been exceeded.",
+                " Time Limit for editing the message has been exceeded.",
+            ),
+            (
+                {"sender_id": 1, "timestamp": 45},
+                False,
+                True,
+                60,
+                False,
+                " Editing sent message is disabled.",
+                " Editing sent message is disabled.",
+            ),
+            ({"sender_id": 1, "timestamp": 45}, True, True, 60, True, None, None),
+            ({"sender_id": 1, "timestamp": 1}, True, True, 0, True, None, None),
         ],
         ids=[
             "msg_sent_by_other_user",
```

Comment:
With more parametrize fields and the next commit adding more cases, it seems good to inline the ids first before the current 2nd commit.